test/services/AuthorityUtilsSpec.scala

```diff
@@ -93,6 +96,14 @@ class AuthorityUtilsSpec extends UnitSpec with MockitoSugar {
     }
   }
 
+  "checkNino" should {
+
+    "return true when a PAYE User has a matching nino" in new TestService {
```

Comment:
False test case?

Reply:
I think we introduced this test when we were learning how to use Auth (TDD child test). We can get rid of this now...